mentorup/src/components/Dashboard/Dashboard.js

```diff
@@ -14,30 +14,37 @@ import {
 
 const Dashboard = () => {
   const { data: user } = useUser();
+  //If user is undefined, uses the local storage
+  const currentUserInfo = user ? user : JSON.parse(localStorage.getItem("currentUserInfo"));
```

Comment:
If you check the other components, you will see that when the user is null, it means that they are unauthenticated. This is a relevant case that needs to be handled properly as part of this component. 

If you try this scenario:
1. Open an incognito window (no session state)
2. Browse directly to the URL for the dashboard

What happens?

It will most likely break because nothing will exist in local storage either, since no cookie has been defined.

Try looking at either the contact, or the user details components, and study the auth code. Then try and replicate that here.